Performed queries and updates to table

mnaibei opened the pull request:
- Performed query and updating of data in the table
- Added a new column to the table.
<img width="1220" alt="Screenshot 2023-06-06 at 12 47 23" src="https://github.com/mnaibei/vet_clinic_database/assets/69506217/fef82e0b-b710-4012-8fa7-527f63c3bf7e">
<img width="1218" alt="Screenshot 2023-06-06 at 12 37 13" src="https://github.com/mnaibei/vet_clinic_database/assets/69506217/87cc2913-20dd-4487-b14f-e7c704f4e12f">
<img width="1220" alt="Screenshot 2023-06-06 at 12 29 58" src="https://github.com/mnaibei/vet_clinic_database/assets/69506217/8a7604fd-46e3-447c-8317-cf00c5299aa9">
<img width="1216" alt="Screenshot 2023-06-06 at 12 28 49" src="https://github.com/mnaibei/vet_clinic_database/assets/69506217/0febf6d3-e932-4f94-bcf9-28182f325fbb">
<img width="1214" alt="Screenshot 2023-06-06 at 12 25 49" src="https://github.com/mnaibei/vet_clinic_database/assets/69506217/c29854e7-a7f9-4644-ad89-3552fc733337">

<img width="605" alt="Screenshot 2023-06-06 at 16 55 26" src="https://github.com/mnaibei/vet_clinic_database/assets/69506217/44a25f03-d834-470d-af0f-6e98f4e6368a">

<img width="782" alt="Screenshot 2023-06-06 at 16 37 52" src="https://github.com/mnaibei/vet_clinic_database/assets/69506217/4fc4f554-ffb9-45a8-ba3b-07250e7779d9">

mnaibei committed:
Performed queries and updates to table

Davidosky007 commented on data.sql:
```diff
@@ -1,7 +1,14 @@
+('Plantmom', '2021-11-15', 2, true,-5.7),
+('Squirtle', '1993-04-02', 3, false, -12.3), 
+('Angemon', '2005-07-12', 1, true, -45), 
+('Boarmon', '2005-07-07', 7, true, 20.04),
```
* Kindly correct the date of birth for the animals( Angemon, Boarmon) its june and not july 
![image](https://github.com/mnaibei/vet_clinic_database/assets/52497246/3c3141e9-c596-4589-b445-fae194a0df5c)

Davidosky007 commented on queries.sql:
```diff
@@ -8,4 +8,71 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```
* As required in the [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) you are to 

> update the animals table by setting the species column to unspecified

> Verify that the change was made on the animal table 

> Then roll back the change and verify that the species columns went back to the state before the transaction. Kindly work on all these 🙏

Davidosky007 commented on queries.sql:
```diff
@@ -8,4 +8,71 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```
* As stated in the requirement 

> verify that the change was made after updating animals' table 

> verify that change persists after commit, Kindly work on these 🙏

mnaibei replied:
@Davidosky007 I have included screenshots for this above. What exactly do you mean by this?

Davidosky007 replied:
According to the requirements you are to write queries that addresses those statements and also add a screenshot of your results

mnaibei replied:
I have done that! the second screenshot from the bottom???

Davidosky007 replied:
@mnaibei yes I can see that but you didn't do it also in your transactions

mnaibei replied:
@Davidosky007 I do not agree with this review and I have asked for a second opinion.

Davidosky007 commented on queries.sql:
```diff
@@ -8,4 +8,71 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
```
* As stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Take%20a%20screenshot%20of%20the,the%20results%20of%20your%20actions.) 

> you are to delete all records in the animals table, then roll back the transaction.
> After the rollback verify if all records in the animals table still exists. below is an example 👇  
```
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```

mnaibei replied:
@Davidosky007 I Have also done this and uploaded the screenshot!

Davidosky007 reviewed (changes requested):
Hi @mnaibei 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Database modified using queries 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mnaibei committed:
Made requested changes

medaminedev66 reviewed (commented):
Hello @mnaibei 👋,

_This is an additional review as a result of the second opinion request._

The CR was right about the first requested change. The date of birth for Angemon and Boarmon is June not July.

Regarding to the requests about transactions, I confirm that you run the transactions correctly on your screenshots. You only need to add those queries of verifying that the changes were made to the transactions in queries.sql file:

Ex: queries.sql file
```sql
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; 
```
Please remember the requirement and its instructions regarding that matter:
<img width="749" alt="image" src="https://github.com/mnaibei/vet_clinic_database/assets/77190203/53b2aeaf-faf3-4d6f-804b-5220ec31435e">

<img width="544" alt="image" src="https://github.com/mnaibei/vet_clinic_database/assets/77190203/37b8137d-1c8f-4eae-9a90-d6e9deb791ec">

Keep rocking 🚀!

mnaibei commented:
@medaminedev66 The first change I agree and I have implemented.
I have also adjusted the parts you have highlighted and uploaded new screenshots on the same.

mnaibei committed:
Made requested changes

Kamba56 reviewed (approved):
Hi @mnaibei, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Great job using transactions to make updates!👏👏👏
- [x] Great job adding screenshots to queries!👏👏👏
- [x]!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
